dashboard/src/components/TimeseriesChart.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState, useMemo } from 'react'
+
+// Libs
+// @ts-ignore
+import { Chart } from 'react-charts'
```

Comment:
I think this has  types definition support please check this. Also did you bench mark this ? I did some benchmarks some time ago and I found Plotly.js to be the best for large data sets, make sure this library will work with the amount of data we will have

Reply:
@senior7515 can we have some numbers to better address this limitation  ?

Reply:
I'm reviewing this library and there are 2 major notes I have on this:

1. it is a beta library and is a bit inactive (last commit 2 months ago) @0x5d what do you think of this?
2. it uses SVG to draw the plots which it should put its limits near other similar libraries at around 30k points, make sure this is enough

Reply:
if possible, we should use the same. the first release can be limited to 1K points per graph. assume 5-10 graphs per page